app/src/viewmodels/camera/CameraViewModel.tsx

```diff
@@ -137,17 +135,18 @@ export default function useCameraViewModel(
         }
 
         // Check if the location is within the group's area
-        const group: DBGroup = await firestoreCtrl.getGroup(group_id);
+        const group: DBGroup = await getGroup(group_id);
         if (!isInGroupArea(location, group)) {
           alert("You need to be in the group's area to create a challenge");
           navigation.navigate("GroupScreen", { currentGroup: group });
           return;
         }
       }
 
-      const imageId = await firestoreCtrl.uploadImage(picture?.uri);
+      console.log("Picture URI: ", picture?.uri);
+      const imageId = await uploadImage(picture?.uri);
+      console.log("image id making challenge: ", imageId);
```

Comment:
You should probably remove those debug logs!

Reply:
needed for the Memories screen